tests/stubtest_whitelists/linux.txt

```diff
@@ -53,5 +52,5 @@ fcntl.[A-Z0-9_]+
 os.SCHED_[A-Z_]+
 
 # Loadable SQLite extensions are disabled on GitHub runners
-sqlite3(.dbapi2)?.Connection.enable_load_extension
-sqlite3(.dbapi2)?.Connection.load_extension
+(sqlite3(.dbapi2)?.Connection.enable_load_extension)?
```

Comment:
Github has decided to have these after all. They show up as noise on my system too, hence I optionally-whitelisted them.